internal/resource/common/defaultref/default_ref_schema.go

```diff
@@ -0,0 +1,15 @@
+package defaultref
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/provider/schema"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/resource/common/resourcelink"
+)
+
+func ToSchema() schema.Attribute {
```

Comment:
I'd make description a parameter to this, so that they don't all have to have the same description

Reply:
this is now in